bin/create-graphql-server.js

```diff
@@ -8,29 +8,348 @@ import cpr from 'cpr';
 import minimist from 'minimist';
 import generate from '../generate';
 import child_process from 'child_process';
+import escapeStringRegexp from 'escape-string-regexp';
+import md5 from 'md5';
+import chalk from 'chalk';
 
 const argv = minimist(process.argv.slice(2));
-
 const commands = argv._;
-
 const SKEL_DIR = path.join(__dirname, '..', 'skel');
+const CHECKSUM_FILE = path.join(__dirname, 'GraphQL.checksums');
```

Comment:
Also this is relative to the script (`__dirname`). I think you want relative to the cwd.